monitoring/influxdb.go

```diff
@@ -30,22 +31,46 @@ func NewInfluxdbMonitor(config InfluxMonitorConfig, logger log.Logger) (Monitor,
 		return nil, errors.Wrapf(err, "couldn't parse influxdb url %v", monitorURL)
 	} else if !u.IsAbs() {
 		return nil, errors.Errorf("influxdb monitoring url %v not absolute url", monitorURL)
+	} else if u.Scheme != "http" && u.Scheme != "https" {
```

Comment:
This check already exists in the InfluxDB client:

https://github.com/influxdata/influxdb/blob/531dc497177bd423b539db35cc0a48b36c4a280c/client/v2/client.go#L95